tests/src/test/scala/whisk/core/entity/test/ExecManifestTests.scala

```diff
@@ -170,4 +176,22 @@ class ExecManifestTests
                     image.localImageName("r", "p", Some("tag")) shouldBe s"r/p/$name:tag"
             }
     }
+
+    it should "throw an error when configured manifest is invalid" in {
+        val config_manifest = "{\"nodejs\":[{\"kind\":\"nodejs\",\"image\":{\"name\":\"nodejsaction\"},\"deprecated\":true},{\"kind\":\"nodejs:6\",\"default\":true,\"image\":{\"name\":\"nodejs6action\"}}]}"
```

Comment:
You can use triple-quotes to prevent escaping in the inner JSON. You could also explore constructing the JSON via scala objects to make this more readable:

```
JsObject(
    "nodejs" -> List(
        JsObject(
            "kind" -> "nodejs".toJson,
            "image" -> JsObject("name" -> "nodejsaction".toJson)
            "deprecated" -> true.toJson
        ),
        JsObject(
            "kind" -> "nodejs:6".toJson,
            "image" -> JsObject("name" -> "nodejs6action".toJson)
            "default" -> true.toJson
        )
    ).toJson
)
```

You can then generate a string out of that using `obj.compactPrint` and write that to the file.

On the same note, we could use a more minimalistic config-manifest as the error is not in the manifest itself, is it?